inc/c_util/channel.h

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef CHANNEL_H
+#define CHANNEL_H
+
+#include "macro_utils/macro_utils.h"
+
+#include "c_pal/gballoc_hl.h"
+#include "c_pal/gballoc_hl_redirect.h"
+#include "c_pal/thandle.h"
+
+#include "c_util/async_op.h"
+#include "c_util/rc_ptr.h"
+
+#define CHANNEL_RESULT_VALUES \
+    CHANNEL_RESULT_SYNC, \
```

Comment:
>SYNC [](http://example.com/codeflow?start=19&length=4)

What does this mean? Does this mean COMPLETED_SYNC?

Reply:
Yes